src/helm/benchmark/adaptation/adapters/multimodal/multimodal_prompt.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+from typing import List
+
+from helm.common.media_object import MediaObject, add_textual_prefix
+
+
+@dataclass(frozen=True)
+class MultimodalPrompt:
+    """Result of multimodal prompt construction."""
+
+    # Global prefix, carried over from `AdapterSpec`
+    global_prefix: str
+
+    # Instance prefix, carried over from `AdapterSpec`. What goes between the instruction and instances.
+    instance_prefix: str
+
+    # Instructions for the task
+    instructions_block: str
+
+    # Train instance blocks for the prompt
+    train_instance_blocks: List[List[MediaObject]]
```

Comment:
I think we should abstract `List[MediaObject]` into a separate dataclass because it shows up multiple times and it is semantically a thing - maybe `MultimediaObject` or something?

Reply:
That makes sense. I had `MultimediaContent` before, but I like the name `MultimediaObject` to be consistent with `MediaObject`.

Reply:
I updated the code and PR description above.